runtime/bcverify/rtverify.c

```diff
@@ -2608,11 +2622,21 @@ j9rtv_verifyArguments (J9BytecodeVerificationData *verifyData, J9UTF8 * utf8stri
 		}
 
 		/* Object or array */
-		if ((*signature == 'L') || arity) {
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+		if ((*signature == 'L') || arity || (*signature == 'Q'))
```

Comment:
define a macro for q or L's

somthing like 
```
#ifdef VT
J9_IS_OBJECT_TYPE(char) ... //Q and L
#else
J9_IS_OBJECT_TYPE(char) ... //Just L
#endif
```

Reply:
Used existing IS_REF_OR_VAL_SIGNATURE let me know if that's fine.

Reply:
yes thats fine